tests/xrt/02_simple/main.cpp

```diff
@@ -20,33 +20,22 @@
 #include <string>
 #include <cstring>
 #include <sys/mman.h>
-#include <time.h>
-#include <chrono>
-#include <thread>
 
-// driver includes
+// XRT includes
+#include "xrt.h"
 #include "ert.h"
-
-// host_src includes
-#include "xclhal2.h"
+#include "experimental/xrt_kernel.h"
+#include "experimental/xrt_xclbin.h"
 #include "xclbin.h"
 
 // lowlevel common include
 #include "utils.h"
 
-#if defined(DSA64)
-#include "xsimple_hw_64.h"     
-#else
-#include "xsimple_hw.h"  
-#endif
-
-static const int count = 1024;
-int foo;
+// This value is shared with worgroup size in kernel.cl
+static const int COUNT = 1024;
```

Comment:
It's much better to use lower case.    You run a serious risk of preprocessor playing tricks with the code.   I think we need to mandate lower case for everything but #defines which all should be proceeded with // anyway.